### DISCUSSION TITLE
Client - Connections list - Funbction to export/imprt connections

### DISCUSSION THREAD

**Discussion by bovirus** [rustdesk/rustdesk#5067] [Feature Request]
@rustdesk 

Please consider to add an option to export/import connection list.

Very useful if you have tor einstall a Window PC to avoid to entsr manuyally all datas (ID/name/password).

Thanks.

**Comment by dinger1986** [comment]
this is covered with the addressbook with pro

**Comment by bovirus** [comment]
@dinger1986 
Thanks.
How can get/pay for pro version?

**Reply by dinger1986** [reply]
https://rustdesk.com/pricing.html

**Comment by bovirus** [comment]
@dinger1986 
I'm not interested about server.
My request is about client.

**Reply by dinger1986** [reply]
Its the server that does the addressbook
